### PR TITLE
Use capital case

### PR DESCRIPTION
**Pull request by eliperkins**
This tweaks the README!

### CODE REVIEW

**Comment by elisburner** [comment]
@eliperkins test

**Comment by elisburner** [comment]
@eliperkins one more time

**Review by eliperkins — COMMENTED** [review]
Wat

**Review by eliperkins — COMMENTED** [review]
Wat

**Review by eliperkins — COMMENTED** [review]
Wat

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# testing
+# Testing
```
Comment (eliperkins) [inline README.md:1]: Wat

**Review by eliperkins — COMMENTED** [review]
Test

**Review by eliperkins — COMMENTED** [review]
Works

**Review by eliperkins — COMMENTED** [review]
Yeahh

**Comment by elisburner** [comment]
@eliperkins pls fix this ty

**Comment by elisburner** [comment]
@eliperkins testing one more time pls

**Comment by elisburner** [comment]
@eliperkins  pls work!

**Comment by elisburner** [comment]
@eliperkins this will work for sure

**Comment by elisburner** [comment]
@eliperkins  pls look

**Comment by elisburner** [comment]
@eliperkins check this

**Comment by elisburner** [comment]
@eliperkins let's goooo

**Comment by elisburner** [comment]
@eliperkins are we good to go?!

**Comment by elisburner** [comment]
@eliperkins testing this one

**Comment by elisburner** [comment]
@eliperkins can I live?!

**Comment by elisburner** [comment]
@eliperkins plssss

File: README.md (hunk 1 above)
Comment (elisburner) [inline README.md:1]: @eliperkins this doesn't work, right?

**Comment by elisburner** [comment]
@eliperkins this one?

**Comment by elisburner** [comment]
@eliperkins  how does this look now?

**Pull request closed without merging**